build/dependencies.props

```diff
@@ -23,6 +23,7 @@
     <MicrosoftAspNetCoreIdentityUIPackageVersion>2.1.3-rtm-30913</MicrosoftAspNetCoreIdentityUIPackageVersion>
     <MicrosoftAspNetCoreMvcPackageVersion>2.1.2-rtm-30913</MicrosoftAspNetCoreMvcPackageVersion>
     <MicrosoftAspNetCoreMvcRazorViewCompilationPackageVersion>2.1.1</MicrosoftAspNetCoreMvcRazorViewCompilationPackageVersion>
+    <MicrosoftAspNetCoreRazorDesignPackageVersion>2.1.1</MicrosoftAspNetCoreRazorDesignPackageVersion>
```

Comment:
Will this get replaced with 2.1.3? That's the version that has the fix.

Reply:
Universe will replace the version listed here with its own version. 2.1.3 isn't on the normal feeds, so things fail if I add it here.

Reply:
Ok cool 👍 better safe than sorry